test/project/partials/formats/erb/.gitkeep

```diff
@@ -0,0 +1 @@
+# Resolver breaks when files and dir's have the same name
```

Comment:
Where are the broken tests that specifically look for this? It seems to me reusing an existing test for this is a bad idea.